dist/testbed-support/README.iotlab.md

```diff
@@ -57,9 +58,9 @@ if `IOTLAB_EXP_ID` is not set.
 
 This schedules a new experiment on the FIT IoT-LAB and waits until it enters
 "Running" state. It will request `IOTLAB_NODES` nodes of type `IOTLAB_TYPE`
-for `IOTLAB_DURATION` minutes at site `IOTLAB_SITE`. With `IOTLAB_PHY_NODES`
-it is possible to choose specific nodes for this experiment by using the resource id
-string format as described above.
+for `IOTLAB_DURATION` minutes at site `IOTLAB_SITE` with `IOTLAB_PROFILE`
```

Comment:
I would suggest to use "using" instead of "with", because the next sentence begins again with "WITH".